### ISSUE TITLE
Some texture colors are off on powerpc

### ISSUE THREAD

**Issue by julianaito** [widelands/widelands#3887]
Hi,

**Describe the bug**
Some texture colors are off on my powerpc machine, as you can see in the screenshot below.

If you don't have the hardware but know how to fix it, feel free to me send me patches :)

**Screenshots**
![widelands](https://user-images.githubusercontent.com/41287367/80794458-41991f80-8b9a-11ea-9a4f-314920e6b35b.png)

**Version:**
- OS: OpenBSD/macppc 6.7-beta
- Compiler: clang 8.0.1
- Widelands: 20
- Machine: PowerBook G4
- GL infos:
```
OpenGL vendor string: X.Org R300 Project
OpenGL renderer string: ATI RV350
OpenGL version string: 2.1 Mesa 19.2.8
OpenGL shading language version string: 1.20
OpenGL extensions:
OpenGL ES profile version string: OpenGL ES 2.0 Mesa 19.2.8
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 1.0.16
OpenGL ES profile extensions:
```

**Comment by klaus-halfmann** [comment]
Wow, you have a _really_ interesting piece of Hardware (I had such an orange Power book years ago). I fear that this Bug is related to some of the OpenGL Libraries (http://www.libsdl.org/) we use together with openBSD. Lets try to find someone with OpenBSD on Intel perhaps to check this.

**Comment by klaus-halfmann** [comment]
Talked to a BSD affine frieend of mine (using FreeBSD) BSD is know to have Issues with openGL. The next bast thing we could do is to locate a FreeBSD Image for Virtual Box and try that one. I myself had a SuSE and then debain lunux on that PPC and it tokk the linux community some years to get that special GPU going.  Please start widelands on a console and give us the output from the start. @Noordfrees / @gunchleoc  any better ideas?

**Comment by julianaito** [comment]
Widelands runs fine on my OpenBSD/amd64 machine, which uses the same [radeon(4) X11 driver](https://man.openbsd.org/radeon.4).

Here is the output log on my PowerBook:
```
This is Widelands Version build-20 (Release)
Set home directory: /home/charlene/.widelands
Adding directory: /usr/local/share/widelands
selected language: (system language)
Graphics: Try to set Videomode 800x600
Graphics: OpenGL: Version "2.1 Mesa 19.2.8"
Graphics: SDL_GL_RED_SIZE is 8
Graphics: SDL_GL_GREEN_SIZE is 8
Graphics: SDL_GL_BLUE_SIZE is 8
Graphics: SDL_GL_ALPHA_SIZE is 0
Graphics: SDL_GL_BUFFER_SIZE is 24
Graphics: SDL_GL_DOUBLEBUFFER is 1
Graphics: SDL_GL_DEPTH_SIZE is 24
Graphics: SDL_GL_STENCIL_SIZE is 0
Graphics: SDL_GL_ACCUM_RED_SIZE is 0
Graphics: SDL_GL_ACCUM_GREEN_SIZE is 0
Graphics: SDL_GL_ACCUM_BLUE_SIZE is 0
Graphics: SDL_GL_ACCUM_ALPHA_SIZE is 0
Graphics: SDL_GL_STEREO is 0
Graphics: SDL_GL_MULTISAMPLEBUFFERS is 0
Graphics: SDL_GL_MULTISAMPLESAMPLES is 0
Graphics: SDL_GL_ACCELERATED_VISUAL is 1
Graphics: SDL_GL_CONTEXT_MAJOR_VERSION is 2
Graphics: SDL_GL_CONTEXT_MINOR_VERSION is 1
Graphics: SDL_GL_CONTEXT_FLAGS is 0
Graphics: SDL_GL_CONTEXT_PROFILE_MASK is 2
Graphics: SDL_GL_SHARE_WITH_CURRENT_CONTEXT is 0
Graphics: SDL_GL_FRAMEBUFFER_SRGB_CAPABLE is 0
Graphics: OpenGL: Double buffering enabled
Graphics: OpenGL: Max texture size: 2048
Graphics: OpenGL: ShadingLanguage: "1.20"
**** GRAPHICS REPORT ****
 VIDEO DRIVER GLVND x11
 pixel fmt 370546692
 size 800 600
**** END GRAPHICS REPORT ****
```

**Comment by Noordfrees** [comment]
This report looks rather like my own machine's graphics summary, except that it has a very old OpenGL version (2.1 and shading language 1.20, I have 4.6 and 4.60). An OpenGL-related problem would be my guess as well.
A VM to simulate this architecture and GL version would be the only way to debug this as far as I can see. But I think that this might be more of a driver/hardware issue so unfortunately I doubt there's much we can do about it :(

**Comment by julianaito** [comment]
That's why i always post OpenGL infos when i find out such issues, it's very likely related to it. Thanks for your quick replies!

**Comment by Niektory** [comment]
I've opened a pull request that might fix it: #3890 

If you could test it that would be great :-)
